lib/puppet/type/mongodb_user.rb

```diff
@@ -52,9 +81,6 @@ def is_to_s(value)
 
   newproperty(:password_hash) do
     desc "The password hash of the user. Use mongodb_password() for creating hash."
-    defaultto do
```

Comment:
this was removed because there can be the case that the plaintext password is present and not the hash, which is acceptable.